.gitignore

```diff
@@ -32,7 +32,7 @@
 
 # Ignore master key for decrypting credentials and more.
 /config/master.key
-
+/config/credentials.yml.enc
```

Comment:
I think we don't have to put this here. Other members might need to update this file.